scipeds/data/queries.py

```diff
@@ -96,7 +96,10 @@ def check_year_range(self):
             warnings.warn(
                 "IPEDS award level coding and race and ethnicity coding changed "
                 "between 2010 and 2011 datasets. For more details, see "
-                "https://nces.ed.gov/ipeds/report-your-data/race-ethnicity-reporting-changes"
+                "https://nces.ed.gov/ipeds/report-your-data/race-ethnicity-reporting-changes. "
+                "Old categories were mapped to new categories according to "
+                "the following published guidance (pg 3): "
+                "https://nces.ed.gov/ipeds/pdf/npec/data/NPEC_Paper_IPEDS_Race_Ethnicity_Deliverable_2012.pdf."
```

Comment:
Instead of linking out to multiple places from the warning, can we link directly to the section of the scipeds FAQ, which can then have more verbose descriptions / link out to wherever?

Reply:
Good call, updated!